recipes/renishaw_wire/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "renishaw_wire" %}
+{% set version = "0.1.15" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+  
+source:
+  url: https://github.com/alchem0x2A/py-wdf-reader/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ada11dec4c472a949b14efa25df006d3c17a100842f2c2ea6f4a61454287121f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - wdf-export=renishawWiRE.export:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.12.0
+    - python >=3.6
+    - matplotlib >=2.1.0
```

Comment:
Unless you need a pyqt GUI, best to depend on `matplotlib-base`.

Reply:
thx! the dependency has been updated